packages/service/tests/unit/grpc/channelpool/test_channel_pool.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from ni_measurement_plugin_sdk_service.grpc.channelpool import GrpcChannelPool
+
+
+@pytest.mark.parametrize(
+    "target,expected_result",
+    [
+        ("127.0.0.1", False),
+        ("[::1]", False),
+        ("localhost", False),
```

Comment:
These first 3 addresses look local, so please add an end-of-line comment to each one saying the port number is missing.